src/config/Config.ts

```diff
@@ -1,11 +1,12 @@
 import fs from 'fs';
 import path from 'path';
+import lodash from 'lodash';
```

Comment:
`lodash` is not a project dependency 😱This only works because some dependencies themselves use `lodash`.

I'm fine with adding it, so do

```sh
yarn add lodash@4.17.15
yarn add --dev @types/lodash
```

And use `camelCase` like so

```ts
import camelCase from 'lodash/camelCase';
```